examples/01-filter/mesh_quality.py

```diff
@@ -6,29 +6,93 @@
 
 Leverage powerful VTK algorithms for computing mesh quality.
 
-Here we will use the :func:`pyvista.DataSetFilters.compute_cell_quality` filter
+Here we will use the :func:`~pyvista.DataObjectFilters.cell_quality` filter
 to compute the cell qualities. For a full list of the various quality metrics
 available, please refer to the documentation for that filter.
+
+Note that many of the measures may only apply to 2D cells or 3D cells. Examples for
```

Comment:
Can we add the Cell Quality Measures table into this example?  It is such a great table and it demonstrates this paragraph at a glance.  I'm fine having it either be dropdown closed or initialized upon open here.